front/app/utils/fileUtils.ts

```diff
@@ -61,16 +63,17 @@ export async function convertUrlToUploadFile(
       response.blob()
     );
     const urlFilename = url.substring(url.lastIndexOf('/') + 1);
-    const uploadFile = convertBlobToFile(
-      blob,
-      filename || urlFilename
-    ) as UploadFile;
-    const base64 = await getBase64FromFile(uploadFile);
-    uploadFile.url = url;
-    uploadFile.base64 = base64;
-    uploadFile.remote = true;
-    uploadFile.filename = filename || urlFilename;
-    uploadFile.id = id || undefined;
+    const file = convertBlobToFile(blob, filename || urlFilename);
+    const base64 = await getBase64FromFile(file);
+    const uploadFile: UploadFile = {
```

Comment:
I prefer to define a const to be as a certain type (`UploadFile`) which makes sure you add all needed attribtues to the object, versus how we did it previously: have a data structure that was not an `UploadFile` but override its type with `as UploadFile`.